Dockerfile

```diff
@@ -2,33 +2,31 @@
 FROM golang:alpine AS builder
 LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"
 
-#golang:alpine set $GOPATH to `/go`
-COPY ./ /go/src/boscoin.io/sebak
-WORKDIR /go/src/boscoin.io/sebak
-
 # You probably don't need to change this
 ARG BUILD_MODE="install"
 ARG BUILD_ARGS=''
 ARG BUILD_PKG="./..."
 
-RUN apk add --no-cache git openssh gcc musl-dev linux-headers
+RUN apk add --no-cache git openssh gcc musl-dev linux-headers ca-certificates
```

Comment:
Why that change ?

Reply:
As described in the above, it is related to building a docker image.
> To improve the performance of building an image, Copy source code should be located bottom as possible. And apk add should be located on top. It may increase CI spead

Reply:
Yes the `COPY/WORKDIR` move makes sense, but my comment is on adding `ca-certificates` here.
The first image is used for building, the second for running.
`ca-certificates` is used for running sebak, not for building it.